app/Services/User/DeleteUserService.php

```diff
@@ -19,7 +19,7 @@ public function __construct(UserRepositoryInterface $userRepository)
     public function handle()
     {
         try {
-            $this->userRepository->delete($this->data->id);
+            $this->userRepository->delete($this->data);
```

Comment:
Đối với những data quan trọng (thông tin user, thông tin thanh toán, ....) khi thực hiện delete thì nên sử dụng Soft Delete, em research thêm phần này nhé